internal/messaging/response_handler.go

```diff
@@ -121,18 +135,22 @@ func (h *evmResponseHandler) handleMintResponse(ctx context.Context, response *R
 
 	currentTime := time.Now()
 
-	if response.MintResponse.BuyableUntil == nil || response.MintResponse.BuyableUntil.Seconds == 0 {
+	switch {
```

Comment:
The switch case uses a few magic numbers (300,70,600) and some are used multiple times.
Would be better to make them constants or at least variables in the scope of the function so that they're a little bit less "magic"